packages/app/lib/src/features/home/data/post_repository.dart

```diff
@@ -108,7 +109,7 @@ final class _AppwritePostRepository implements PostRepository {
           'author': author,
           'lat': lat,
           'lng': lng,
-          'timestamp': DateTime.timestamp(),
+          'timestamp': DateFormat('yyyy-MM-ddTHH:mm:ss').format(DateTime.now()),
```

Comment:
I still feel an extension type would be more robust.

Reply:
On second thought, I forgot about [`JsonConverter`s](https://github.com/google/json_serializable.dart/blob/3e4dbe3ff6b7989e99e835a6533ce6fcfb2a8e51/example/lib/json_converter_example.dart#L26-L34), we should swap out the unneeded extension types with them.

Reply:
Also, DateTime.now isn't in UTC.